modules/programs/atuin.nix

```diff
@@ -130,7 +130,7 @@ in {
         if not ($atuin_cache | path exists) {
           mkdir $atuin_cache
         }
-        ${cfg.package}/bin/atuin init nu ${flagsStr} | save --force ${config.xdg.cacheHome}/atuin/init.nu
+        ${cfg.package}/bin/atuin init nu ${flagsStr} | str replace --string --all 'let-env ' '$env.' | save --force ${config.xdg.cacheHome}/atuin/init.nu
```

Comment:
I'd rather this be fixed in nixpkgs by applying https://github.com/atuinsh/atuin/pull/1080